### ISSUE TITLE
Unrecognized areas

### ISSUE THREAD

**Issue by bropines** [dmMaze/BallonsTranslator#335]
There are some pictures in which you don't want anything to be recognized. Like an author's signature, or a watermark. Is it possible to create a layer that will tell the OCD or Text detector not to detect here, move on?
Like the text at the bottom of this picture.
![image](https://github.com/dmMaze/BallonsTranslator/assets/57861007/a83ebcf5-e8dd-4f45-a828-aa263d802003)

**Comment by lhj5426** [comment]
你可以先 OCR一下  然后把这个OCR结果 写入 OCR 替换 并启动 去掉无字框
这样就 没事  临时解决方案

**Comment by bropines** [comment]
> 你可以先 OCR一下 然后把这个OCR结果 写入 OCR 替换 并启动 去掉无字框
> 这样就 没事 临时解决方案

That’s what I did, but sometimes it recognizes differently and I can’t process all the options like that, plus the load on the PC will drop if I just make ignore areas

**Comment by Mistermozg1** [comment]
> 你可以先 OCR一下 然后把这个OCR结果 写入 OCR 替换 并启动 去掉无字框 这样就 没事 临时解决方案

but the frames only remove what's around the text, and in the original image the text is larger than my font, so there's still ugly framing and chunks from the new grout layer

**Comment by Mistermozg1** [comment]
> > 你可以先 OCR一下 然后把这个OCR结果 写入 OCR 替换 并启动 去掉无字框
> > 这样就 没事 临时解决方案
> 
> That’s what I did, but sometimes it recognizes differently and I can’t process all the options like that, plus the load on the PC will drop if just make ignore areas

Yes! this is the perfect option, because there will be no load on the pc, and no unnecessary waste on deepl pro. the characters go away very quickly

**Comment by lhj5426** [comment]
![image](https://github.com/dmMaze/BallonsTranslator/assets/30548000/ea939674-3f96-43cd-950f-84deadc79b6f)

![image](https://github.com/dmMaze/BallonsTranslator/assets/30548000/247b264b-1e38-4d02-a361-f499bcb86963)

就想这个样子 可以期待一下 哈哈

**Comment by bropines** [comment]
> 就想这个样子 可以期待一下 哈哈

Hmmm. А paddleOCR и RapidOCR неплохо так распознают английский и китайский, а вот с японским безбожно сосут...
Chinese:
![image](https://github.com/dmMaze/BallonsTranslator/assets/57861007/07e610ad-3086-4963-baf6-370af4cf1d95)
English:
![image](https://github.com/dmMaze/BallonsTranslator/assets/57861007/4e620d44-559b-4572-b484-c3066e58de70)
Japanese:
![9hYIRMRU5a](https://github.com/dmMaze/BallonsTranslator/assets/57861007/f9c2ae4c-0e70-4e9f-83b4-f8e230688674)

**Comment by lhj5426** [comment]
> > 就想这个样子 可以期待一下 哈哈
> 
> Hmmm. PaddleOCR and RapidOCR recognize_ English _和 _Chinese_ quite well, but they suck at Japanese... _Chinese_: ![image](https://private-user-images.githubusercontent.com/57861007/290222287-07e610ad-3086-4963-baf6-370af4cf1d95.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0Nzg3NDYsIm5iZiI6MTcwMjQ3ODQ0NiwicGF0aCI6Ii81Nzg2MTAwNy8yOTAyMjIyODctMDdlNjEwYWQtMzA4Ni00OTYzLWJhZjYtMzcwYWY0Y2YxZDk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEzVDE0NDA0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThkZDEwNzFhNTYwMzY4YTFmZjNlMzRjNTcxOGRiNjk0NGE3Y2QxY2NmOWI5MWQyZDdiODRjMGFkZWFiZmRjY2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JLz6FoNDIRqy4C-H2SkNymi7tSiL62ZoluTmOBW9zYA) _English_: ![image](https://private-user-images.githubusercontent.com/57861007/290223679-4e620d44-559b-4572-b484-c3066e58de70.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0Nzg3NDYsIm5iZiI6MTcwMjQ3ODQ0NiwicGF0aCI6Ii81Nzg2MTAwNy8yOTAyMjM2NzktNGU2MjBkNDQtNTU5Yi00NTcyLWI0ODQtYzMwNjZlNThkZTcwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEzVDE0NDA0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJlZTc4MGUwMzczNWQ1Y2VmODkwNTg0ZjU5Yjk3MWZkMjkzNWI2MGI3MDBhYzFkMjYzZWFlMWZkZWMyZDAyOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4c0WmIu7-rFk6UuWTeFFsiBqw7SOQvH5MnRk9k4Qo0E) Japanese: ![9hYIRMRU5a](https://private-user-images.githubusercontent.com/57861007/290223793-f9c2ae4c-0e70-4e9f-83b4-f8e230688674.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0Nzg3NDYsIm5iZiI6MTcwMjQ3ODQ0NiwicGF0aCI6Ii81Nzg2MTAwNy8yOTAyMjM3OTMtZjljMmFlNGMtMGU3MC00ZTlmLTgzYjQtZjhlMjMwNjg4Njc0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEzVDE0NDA0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzZTg3YzM3N2VkNjkzZDRjYTFjYzcyMDdjYWFkMDdkNTg1YjFlYzVjN2MwYWE2ZDhiYmRjMDMzN2VlZmNjMGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fVGB2MtciIbjVBspTHlJO8L3izFuQOvqTGLMhCUpre0)

"I mean, can we expect this feature to be added to BallonTranslator instead of using Umi-OCR? The vertical text recognition of Umi-OCR is very poor."
